simple-nft-svelte/README.md

```diff
@@ -0,0 +1,47 @@
+# Svelte + Vite
```

Comment:
Write a simple explanation of what this app does with a link to the tutorial as in https://github.com/trilitech/tutorial-applications/blob/main/nft-taquito/README.md